client/src/pages/Translations.jsx

```diff
@@ -1,17 +1,112 @@
 const { TabPane } = Tabs;
 
+const languages = ['Spanish', 'French', 'Chinese'];
+const priorities = { Urgent: 4, High: 3, Medium: 2, Low: 1 };
+
 function Translations() {
-  const columns = [
+  const [verifiedList, setVerifiedList] = useState({});
+  const [unverifiedList, setUnverifiedList] = useState({});
+
+  const updateLists = useCallback((verifications) => {
+    const verifiedResources = { Spanish: [], French: [], Chinese: [] };
+    const unverifiedResources = { Spanish: [], French: [], Chinese: [] };
+    languages.forEach((l) => {
+      const resources = verifications[l];
+
+      resources?.forEach((verification) => {
+        const percentage =
+          (verification.verifiedTranslationsCount /
+            verification.totalTranslations) *
+          100;
+
+        if (verification.totalReports >= 1) {
+          unverifiedResources[l].push({
+            key: '1',
+            name: verification.name,
+            priority: <PriorityIcon priorityType="Urgent" />,
+            status: <StatusTag status="Unverified" />,
+            translate: (
+              <Link
+                to={
+                  `/translation/${verification._id}?` +
+                  `language=${l}&type=${verification.type}`
+                }
+              >
+                Translate
+              </Link>
+            ),
+          });
+        } else if (percentage === 100.0) {
+          verifiedResources[l].push({
+            key: '1',
+            name: verification.name,
+            status: <StatusTag status="Verified" />,
+            translate: (
+              <Link
+                to={
+                  `/translation/${verification._id}?` +
+                  `language=${l}&type=${verification.type}`
+                }
+              >
+                Translate
+              </Link>
+            ),
+          });
+        } else {
+          let priority;
+          if (percentage < 30) {
+            priority = <PriorityIcon priorityType="High" />;
+          } else if (percentage < 50) {
+            priority = <PriorityIcon priorityType="Medium" />;
+          } else {
+            priority = <PriorityIcon priorityType="Low" />;
+          }
+          unverifiedResources[l].push({
+            key: '1',
+            name: verification.name,
+            priority,
+            status: <StatusTag status="Unverified" />,
+            translate: (
+              <Link
+                to={
+                  `/translation/${verification._id}?` +
+                  `language=${l}&type=${verification.type}`
+                }
+              >
+                Translate
+              </Link>
+            ),
+          });
+        }
+      });
+    });
+    setVerifiedList(verifiedResources);
+    setUnverifiedList(unverifiedResources);
+  }, []);
+
+  useEffect(() => {
+    const newVerifications = {};
+    languages.forEach(async (language) => {
+      const res = await getVerifications(language);
+      if (res != null) {
```

Comment:
I think `if (res)` will do the same but also catch other falsy values like undefined, etc.

Reply:
`!=` catches undefined as well!